test/unit/app/renderer/components/preferences/advancedTabTest.js

```diff
@@ -30,49 +27,13 @@ describe('AdvancedTab component', function () {
 
     mockery.registerMock('electron', fakeElectron)
     window.chrome = fakeElectron
-
     AdvancedTab = require('../../../../../../app/renderer/components/preferences/advancedTab')
   }
 
   afterEach(function () {
     mockery.disable()
-  })
-
-  const platformUtilMac = {isLinux: () => false, isWindows: () => false, isDarwin: () => true}
-  const platformUtilLinux = {isLinux: () => true, isWindows: () => false, isDarwin: () => false}
+  }) */
 
   describe('AdvancedTab', function () {
```

Comment:
Thanks for leaving this file in place, I'd imagine we'll get more tests than just that one switch in the future.